ubyssey/helpers.py

```diff
@@ -76,66 +79,44 @@ def insert_ads(content, article_type='desktop'):
         return ad_placements
 
     @staticmethod
-    def get_frontpage(reading_times=None, section=None, section_id=None, sections=[], exclude=[], limit=7, is_published=True, max_days=14):
-
-        if is_published:
-            is_published = 1
-        else:
-            is_published = 0
-
-        if reading_times is None:
-            reading_times = {
-                'morning_start': '9:00:00',
-                'midday_start': '11:00:00',
-                'midday_end': '16:00:00',
-                'evening_start': '16:00:00',
-            }
-
-        context = {
-            'section': section,
-            'section_id': section_id,
-            'excluded': ",".join(map(str, exclude)),
-            'sections': ",".join(sections),
-            'limit': limit,
-            'is_published': is_published,
-            'max_days': max_days
-        }
-
-        context.update(reading_times)
-
-        query = """
-            SELECT *, TIMESTAMPDIFF(SECOND, published_at, NOW()) as age,
-            CASE reading_time
-                 WHEN 'morning' THEN IF( CURTIME() < %(morning_start)s, 1, 0 )
-                 WHEN 'midday'  THEN IF( CURTIME() >= %(midday_start)s AND CURTIME() < %(midday_end)s, 1, 0 )
-                 WHEN 'evening' THEN IF( CURTIME() >= %(evening_start)s, 1, 0 )
-                 ELSE 0.5
-            END as reading,
-            TIMESTAMPDIFF(DAY, published_at, NOW()) <= %(max_days)s as age_deadline
-            FROM dispatch_article
-        """
-
-        query_where = """
-            WHERE head = 1 AND
-            is_published = %(is_published)s AND
-            parent_id NOT IN (%(excluded)s)
-        """
+    def get_frontpage(section=None, sections=[], exclude=[], limit=7, is_published=True, max_days=14):
```

Comment:
Maybe the `section` parameter should just be removed, seeing as it's semantically the same as `sections=[section]`?